### DISCUSSION TITLE
Starstruck badge not reporting the stars count properly

### DISCUSSION THREAD

**Discussion by janfilips** [community/community#20835] [Other Feature Feedback, Questions, & Ideas]
For real - check on my own github account, it says 16 stars whilst I had 69 and 27 forks.  👎

**Comment by radar-blip** [comment]
The Starstruck achievement badge will continue to show 16 until you reach the next tier, which is 128. Then it will change to 128.
Same with some of the other badges also. I believe thats how it works.

**Reply by janfilips** [reply]
Doesn't really compute.
16 -> 32 -> 64 -> 128 does but still the wording on the badge literally says: 

<img width="224" alt="Screenshot 2022-07-19 at 2 59 26" src="https://user-images.githubusercontent.com/870699/179636464-10da10ea-3eab-4f6d-b002-76ece4486f9a.png">

janfilips/blockchain - 16 stars  (whilst the actual count is 69).

I get it but you know, it is a bit misleading, no?

**Reply by radar-blip** [reply]
Yes, I get what you're saying. I was just explaining that it wont update until the next tier is reached. 
The badge itself didn't show up until you reached 16 stars, right?

**Reply by radar-blip** [reply]
Its not misleading, you need 128 for the bronze starstruck achievement, not 69.

**Comment by aashutoshrathi** [comment]
It's based on the first repo reaching the n starts for a level!
So, It'll only report the start count once it reaches one of the levels.

<img width="446" alt="image" src="https://user-images.githubusercontent.com/21199234/180585609-7544ba78-9f92-4060-9afd-816b4497eae0.png">

**Reply by shm007g** [reply]
I seems true. 100 star shows 16. 150 star shows 128.
